pages.zh_TW/common/az.md

```diff
@@ -1,10 +1,10 @@
 # az
 
 > Microsoft Azure 的官方客戶端工具。
-> 此命令也有關於其子命令的文件，例如：`login`.
-> 更多資訊：<https://learn.microsoft.com/cli/azure>.
+> 此命令也有關於其子命令的文件，例如：`login`。
+> 更多資訊：<https://learn.microsoft.com/zh-tw/cli/azure/reference-index>。
```

Comment:
Using Latin period is a style rule of this repo. See:  https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/more-info-link.md#zh_tw
```suggestion
> 更多資訊：<https://learn.microsoft.com/zh-tw/cli/azure/reference-index>.
```